resources/web/templates/x3d_playback.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>%title%</title>
     <link type="text/css" href="https://www.cyberbotics.com/jquery-ui/1.11.4/jquery-ui.min.css" rel="stylesheet"/>
-    <link type="text/css" href="https://www.cyberbotics.com/wwi/R2021b/wwi.css" rel="stylesheet"/>
+    <link type="text/css" href="https://www.cyberbotics.com/wwi/Wrenjs/wwi.css" rel="stylesheet"/>
```

Comment:
This path is ok for testing.
But when in production we need to have a folder for each major version so that previously Webots releases are not broken when applying changes to the develop branch.

Reply:
Indeed, but we needed to share the animation player with the people of robocup so that they can use it and also for testing.
But we should revert it back once #2769 is ready to be merged